src/stories/components/SideNav.stories.tsx

```diff
@@ -0,0 +1,119 @@
+import React from 'react';
+import { ComponentStory, ComponentMeta } from '@storybook/react';
+import { SideNav } from '../../components/SideNav';
```

Comment:
please fix capitalization in `src/components/Sidenav`, my build is saying: 
Should be: `SideNav` folder.
![image](https://user-images.githubusercontent.com/27584221/202774463-6f8dec39-8d44-473f-9696-b9f8a514e989.png)

Reply:
I fixed it